src/main/java/com/kuba6000/mobsinfo/loader/extras/Reliquarry.java

```diff
@@ -0,0 +1,125 @@
+            addDrop(drops, witchHatStack, Names.witch_hat);
+        } else if (recipe.entity instanceof EntitySpider) {
+            addDrop(drops, XRRecipes.ingredient(Reference.SPIDER_INGREDIENT_META), Names.spider_fangs);
+        } else if (recipe.entity instanceof EntityCaveSpider) {
+            addDrop(drops, XRRecipes.ingredient(Reference.SPIDER_INGREDIENT_META), Names.cave_spider_fangs);
+        } else if (recipe.entity instanceof EntitySkeleton) {
+            addDrop(drops, XRRecipes.ingredient(Reference.SKELETON_INGREDIENT_META), Names.rib_bone);
+            if (((EntitySkeleton) recipe.entity).getSkeletonType() == 1) {
+                addDrop(drops, XRRecipes.ingredient(Reference.WITHER_INGREDIENT_META), Names.withered_rib);
+            }
+        } else if (recipe.entity instanceof EntityZombie) {
+            addDrop(drops, XRRecipes.ingredient(Reference.ZOMBIE_INGREDIENT_META), Names.zombie_heart);
+        } else if (recipe.entity instanceof EntityPigZombie) {
+            addDrop(drops, XRRecipes.ingredient(Reference.ZOMBIE_INGREDIENT_META), Names.pigman_heart);
+        } else if (recipe.entity instanceof EntitySlime) {
+            addDrop(drops, XRRecipes.ingredient(Reference.SLIME_INGREDIENT_META), Names.slime_pearl);
+        } else if (recipe.entity instanceof EntityBlaze) {
+            addDrop(drops, XRRecipes.ingredient(Reference.MOLTEN_INGREDIENT_META), Names.blaze_molten_core);
+        } else if (recipe.entity instanceof EntityMagmaCube) {
+            addDrop(drops, XRRecipes.ingredient(Reference.MOLTEN_INGREDIENT_META), Names.magma_cube_molten_core);
+        } else if (recipe.entity instanceof EntityGhast) {
+            addDrop(drops, XRRecipes.ingredient(Reference.CREEPER_INGREDIENT_META), Names.ghast_gland);
+        } else if (recipe.entity instanceof EntityCreeper) {
+            addDrop(drops, XRRecipes.ingredient(Reference.CREEPER_INGREDIENT_META), Names.creeper_gland);
+            if (((EntityCreeper) recipe.entity).getPowered()) {
+                addDrop(drops, XRRecipes.ingredient(Reference.STORM_INGREDIENT_META), Names.eye_of_the_storm);
+            }
+        } else if (recipe.entity instanceof EntityEnderman) {
+            addDrop(drops, XRRecipes.ingredient(Reference.ENDER_INGREDIENT_META), Names.ender_heart);
+        } else if (recipe.entity instanceof EntityBat) {
+            addDrop(drops, XRRecipes.ingredient(Reference.BAT_INGREDIENT_META), Names.bat_wing);
+        } else if (recipe.entity instanceof EntitySnowman) {
+            addDrop(drops, XRRecipes.ingredient(Reference.FROZEN_INGREDIENT_META), Names.frozen_core);
+        }
+    }
+
+    private void addDrop(ArrayList<MobDrop> drops, ItemStack item, String name) {
+        MobDrop drop = new MobDrop(item, MobDrop.DropType.Normal, 0, null, null, false, false);
+        drop.variableChance = true;
+        drop.chanceModifiers.add(new ReliquaryDropChance(name, eventHandler));
```

Comment:
Is there anything special about that drop chance? The only thing I see in the description is the chance. If thats the case, there is no need for that
```suggestion
        MobDrop drop = new MobDrop(item, MobDrop.DropType.Normal, (int)(eventHandler.getBaseDrop(name) * 10000d), null, null, false, false);
```